packages/xo-server/src/api/host.js

```diff
@@ -215,6 +215,21 @@ emergencyShutdownHost.resolve = {
 
 // -------------------------------------------------------------------
 
+export async function isConsistentHostServerTime({ host }) {
+  await this.getXapi(host).assertConsistentHostServerTime(host._xapiRef)
```

Comment:
I think it's not problem if it take error as a 'no' because we need to know if the host is consistent or not and we don't need other errors and even the name of the function `isConsistentHostServerTime` confirms it.

Reply:
I think no problem if the error is considered a "no" because we need to know if the host is consistent or not.